bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -43,8 +43,7 @@ spec:
       tolerations: {{- include "postgresql-ha.tplValue" (dict "value" .Values.pgpool.tolerations "context" $) | nindent 8 }}
       {{- end }}
       {{- if .Values.pgpool.securityContext.enabled }}
-      securityContext:
-        fsGroup: {{ .Values.pgpool.securityContext.fsGroup }}
+      securityContext: {{- omit .Values.pgpool.securityContext "enabled" | toYaml | nindent 8 }}
```

Comment:
From my point, this is a confusing way to select the queys (using negative logic).
I think it would be more readable to follow the previous approach.

This comment applies to the rest of case where you used the `omit` function

Reply:
Hi @djcass44 ,
I talked about this with the team and we are going to accept it since it seems to be the more customizable way to add the securityContext.